### ISSUE TITLE
Can't submit text if you ever write something in the url field

### ISSUE THREAD

**Issue by wodim** [reddit/reddit#1717]
1. https://www.reddit.com/submit
2. write something in the url field, then remove it
3. click the text tab
4. fill all fields correctly
5. click submit; nothing ever happens
6. click the link tab again
7. the error "a url is required" is shown

**Comment by 13steinj** [comment]
Yeah-- seems as though the form for the url / image doesn't get disabled properly when changing to a text post.
